spec/lib/gateway/stat_gateway_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe StatGateway do
+  before do
+    DB[:userdetails].truncate
```

Comment:
Be nice to remove this duplication of truncate at some point, maybe using a truncation gem?

Reply:
Agree.